routes/index.js

```diff
@@ -1,55 +1,473 @@
-var express = require('express');
-var router = express.Router();
-var Song = require('../models/song');
-var User = require('../models/user')
-var app = require('../app')
-var pdf = require('html-pdf')
-var fs = require('file-system')
-
-
-/* GET home page. */
-router.get('/', function(req, res, next) {
-    Song.find(function(err, songs) {
-            Song.find({
-                source: {
-                    $ne: undefined
-                }
-            }, function(err, demo) {
-                console.log(demo)
+var express = require('express'),
+    router = express.Router(),
+    Song = require('../models/song'),
+    User = require('../models/user'),
+    _ = require('lodash'),
+    Playlist = require('../models/playlist'),
+    async = require('async')
+
+
+router.get('/temp', function(req, res){
+  for (var i = 0; i < 20; i++){
+    new Song({
+      title: 'english song ' + i,
```

Comment:
this route is just for adding new field for the existing song. for testing purposes